scripts/validate/check-all-packages-vendored

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eu
+
+FILE_PATH='/tmp/vndr.log'
+WARN_MISS_VENDOR='WARNING: dependency is not vendored'
+
+if grep -q $WARN_MISS_VENDOR $FILE_PATH; then
```

Comment:
We should quote `$FILE_PATH` here, or perhaps we should just hard-code it, because it's not something we really change

Reply:
okay